crates/nu-command/src/filesystem/touch.rs

```diff
@@ -213,21 +213,21 @@ impl Command for Touch {
                 example: "touch -m fixture.json",
                 result: None,
             },
-            Example {
-                description: "Changes the last modified time of files a, b and c to a date",
-                example: r#"touch -m -d "yesterday" a b c"#,
-                result: None,
-            },
+            // Example {
```

Comment:
you can just delete them rather than commenting them out, as git will store the history if we ever need them again.